app/uk/gov/hmrc/apicataloguepublish/apicatalogue/config/ApiCatalogueAdminConnectorConfigProvider.scala

```diff
@@ -14,16 +14,15 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.apiplatformapicataloguepublish.apicatalogue.config
+package uk.gov.hmrc.apicataloguepublish.apicatalogue.config
 
```

Comment:
This package name change no longer matches the repository name. Is that just to make the name shorter?

Reply:
No - it's to avoid apiplatformxxxxx leading text causing package/import issues with the apiplatform package from libs.  I know - it's madness but the alternative was to have to put imports in a specific order or not use wildcards or ... Yuk.

Reply:
Seemed the lesser of 2 evils.